src/ontology/edits/oeo-social.omn

```diff
@@ -883,14 +924,87 @@ Class: OEO_00010461
+    
     SubClassOf: 
         OEO_00140151,
         <http://purl.obolibrary.org/obo/IAO_0000136> some OEO_00010457
     
     
+Class: OEO_00010492
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "An economic instrument function is a realizable entity of a policy instrument that promotes, amongst others, economic measures."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1826
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1906",
+        rdfs:label "economic instrument function"@en
+    
+    SubClassOf: 
+        <http://purl.obolibrary.org/obo/BFO_0000034>,
+        <http://purl.obolibrary.org/obo/RO_0000052> some OEO_00140151
+    
+    
+Class: OEO_00010493
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "An education instrument function is a realizable entity of a policy instrument that promotes, amongst others, education measures."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1826
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1906",
+        rdfs:label "education instrument function"@en
+    
+    SubClassOf: 
+        <http://purl.obolibrary.org/obo/BFO_0000034>,
+        <http://purl.obolibrary.org/obo/RO_0000052> some OEO_00140151
+    
+    
+Class: OEO_00010494
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "An information instrument function is a realizable entity of a policy instrument that promotes, amongst others, information measures."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1826
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1906",
+        rdfs:label "information instrument function"@en
+    
+    SubClassOf: 
+        <http://purl.obolibrary.org/obo/BFO_0000034>,
+        <http://purl.obolibrary.org/obo/RO_0000052> some OEO_00140151
+    
+    
+Class: OEO_00010495
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "An regulatory instrument function is a realizable entity of a policy instrument that promotes, amongst others, regulatory measures.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1826
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1906",
+        rdfs:label "regulatory instrument function"@en
+    
+    SubClassOf: 
+        <http://purl.obolibrary.org/obo/BFO_0000034>,
+        <http://purl.obolibrary.org/obo/RO_0000052> some OEO_00140151
+    
+    
+Class: OEO_00010496
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "An voluntary agreement instrument function is a realizable entity of a policy instrument that promotes, amongst others, voluntary agreement measures."@en,
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "An voluntary agreement instrument function is a function of a policy instrument that promotes, amongst others, voluntary agreement measures."@en,
```

Reply:
We could think about an intermediate class `policy instrument function` to group these new functions. But I am also ok with the current state.

Reply:
I thought about that, too. However, I did not find a good definition proposal.

Also: as we now don't use the `has function` resp. `function of` I am thinking whether we should relabel the classes to `economic instrument characteristic` etc.

Reply:
We classify them as functions, which still makes sense to me. There is no `characteristic` entity. Therefore I'd prefer to keep the label.